### ISSUE TITLE
[Site-Frontend] Utilisation d'un CMS

### ISSUE THREAD

**Issue by Sazaju** [Sazaju/Zero-Fansub-website#28]
Les CMS apportent de nombreuses fonctionnalités, et il serait bien d'en avoir un spécialisé pour le fansub (c'est le but du projet [FSP](https://github.com/Sazaju/FSP)). Cela dit, si dans une moindre mesure certains CMS existants permettent déjà d'obtenir un grand nombre de fonctionnalités exploitables pour du fansub, on peut imaginer les utiliser, voire les étendre s'ils sont open source.

À considérer :
- XWiki
- Drupal
- Drupal Gardens
- Django
- EZ Publish
- Oracle WCS

**Comment by thake-viral** [comment]
Cual de estos CMS funciona con el script Zero-Fansub-website

**Comment by Sazaju** [comment]
None of them. This issue is about considering CMSs to simplify our work, but nothing has been done in this direction so far. I am the only programer in the team, and I am of the kind "all from scratch", so I rarely use anything but the basics. This issue is mainly here as a reminder, just in case. Most of the issues are so, actually.

**Comment by thake-viral** [comment]
Thanks for the answer, in any case there is no way to add some post, projects, etc ...

**Comment by Sazaju** [comment]
Yes, you should check out the BDD branch for that. But it is not finalized. In the master branch it should be added by code.

**Comment by thake-viral** [comment]
Just wait.

**Comment by thake-viral** [comment]
And you could not upload it so as to prove full truth I really like your
work. I would be very grateful his climb . And thanks for the reply .

2016-05-23 15:49 GMT-05:00 Sazaju HITOKAGE <notifications@github.com>:
